### ISSUE TITLE
Implement additional python code formatting checks

### ISSUE THREAD

**Issue by wlach** [iodide-project/iodide#1552]
I've been really happy with using prettier to lint/check javascript. If you integrate it with your editor to reformat-on-save, you can basically say goodbye to worrying about formatting issues *at all*.

I'd like us to consider doing the same thing with our python codebase. The equivalent tool there seems to be [black](https://github.com/ambv/black) which covers everything but header declaration order (which is provided by [isort](https://pypi.org/p/isort/))

Basically two steps here:

1. Reformat all code with isort and black
2. Set up ci to automatically check that new code conforms to their standards

@robhudson expressed some interest in doing this (at least the isort part, but black would be a logical extension).

**Comment by hamilton** [comment]
:tada: all for it.

**Comment by robhudson** [comment]
I'm all for it and happy to work on these changes.
